feature-libs/checkout/scheduled-replenishment/components/replenishment-order-confirmation.module.ts

```diff
@@ -0,0 +1,74 @@
+import { CommonModule } from '@angular/common';
+import { NgModule, Type } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import {
+  OrderConfirmationGuard,
+  OrderConfirmationItemsComponent,
+  OrderConfirmationModule,
+  OrderConfirmationOverviewComponent,
+  OrderConfirmationTotalsComponent,
+} from '@spartacus/checkout/base/components';
+import {
+  CmsConfig,
+  FeaturesConfigModule,
+  I18nModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import {
+  CardModule,
+  CartSharedModule,
+  FormErrorsModule,
+  PromotionsModule,
+  PwaModule,
+} from '@spartacus/storefront';
+import { ScheduledReplenishmentOrderConfirmationThankYouMessageComponent } from './order-confirmation-thank-you-message/order-confirmation-thank-you-message.component';
+
+const orderConfirmationComponents: Type<any>[] = [
+  ScheduledReplenishmentOrderConfirmationThankYouMessageComponent,
+];
+
+@NgModule({
+  imports: [
+    CommonModule,
+    CartSharedModule,
+    CardModule,
+    PwaModule,
+    PromotionsModule,
+    I18nModule,
+    ReactiveFormsModule,
+    FeaturesConfigModule,
+    FormErrorsModule,
+    OrderConfirmationModule,
+  ],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        OrderConfirmationThankMessageComponent: {
+          component:
+            ScheduledReplenishmentOrderConfirmationThankYouMessageComponent,
+          guards: [OrderConfirmationGuard],
+        },
+        ReplenishmentConfirmationMessageComponent: {
+          component:
+            ScheduledReplenishmentOrderConfirmationThankYouMessageComponent,
+          guards: [OrderConfirmationGuard],
+        },
+        ReplenishmentConfirmationOverviewComponent: {
+          component: OrderConfirmationOverviewComponent,
+          guards: [OrderConfirmationGuard],
+        },
+        ReplenishmentConfirmationItemsComponent: {
+          component: OrderConfirmationItemsComponent,
+          guards: [OrderConfirmationGuard],
+        },
+        ReplenishmentConfirmationTotalsComponent: {
+          component: OrderConfirmationTotalsComponent,
+          guards: [OrderConfirmationGuard],
+        },
+      },
+    }),
+  ],
+  declarations: [...orderConfirmationComponents],
+  exports: [...orderConfirmationComponents],
+})
+export class ScheduledReplenishmentOrderConfirmationModule {}
```

Comment:
let's move this file under `feature-libs/checkout/scheduled-replenishment/components/order-confirmation-thank-you-message` folder

Reply:
Hmm, but this module maps CMS components for all repl components, so I think it's OK for it to stay here.
Maybe I misunderstood you?

Reply:
got it~ 

I think I was trying to align the folders from base and here, so maybe it would be best to have the order-confirmation folder